docs/checkit.js

```diff
@@ -91,6 +88,42 @@ function CheckIt() {
 
 };
 
+CheckIt.prototype.attachCellClickHandler = function(calObj) {
+    
+    var checkitObj = this;
+    this.$calendarDiv.find('.cell').click(function(event) {
```

Comment:
Note that this is going to have to change or it will attach additional handlers when you try to add months to the page.